router/core/operation_processor.go

```diff
@@ -610,6 +625,10 @@ func (o *OperationKit) normalizePersistedOperation() (cached bool, err error) {
 	o.parsedOperation.NormalizedRepresentation = o.kit.normalizedOperation.String()
 	o.parsedOperation.Request.Variables = o.kit.doc.Input.Variables
 
+	if o.operationProcessor.persistedOperationClient != nil && o.operationProcessor.persistedOperationClient.ApqEnabled() {
```

Comment:
This was meant to ensure that in cases where we're using APQ, the requests are cached in the proper location, rather than in the general persisted operation cache. However, this could also potentially mean that if a user has both persisted operations enabled and apq enabled, and they requested an operation from their file cached persisted operations, it would get cached in the local cache (because by this point in the operation, we don't know whether the request came from APQ or persisted operation). I welcome any suggestions for how we should address this, if it should be done at this point

Reply:
I think it's ok if you use both caches, the distributed (Redis) cache to load the APQ content, and the PO cache to cache the normalized version of the APQ. The naming might be slightly off, but I think it'll work fine. Can you remove this early return and validate if tests behave as expected?

I think it's important to cache this step also for APQ because otherwise APQ would never hit the normalization cache, right?

Again, I'm not 100% sure so please test and verify.

Reply:
The one case that this causes issues with is `ttl expiration`, because we can't enforce that the KV will drop the key when the TTL is over. Does that matter?